doc/subsystems/disk/sdhc.rst

```diff
@@ -0,0 +1,42 @@
+SDHC disks
+##########
+
+Zephyr includes support for connecting a SDHC card via the SPI bus.
+When used with ELMFS, this allows reading and writing FAT based
+filesystems.
+
+The system has been tested with cards from Samsung, SanDisk, and 4V
+with sizes from 2 GiB to 32 GiB in single partition mode.
+
+MMC and SDSC (<= 2 GiB) cards are not supported and will be ignored.
```

Comment:
... and cards > 32GB are ignored too?

Reply:
Clarified that it should work for higher capacity but hasn't been tested.